cmds/removeroles.js

```diff
@@ -0,0 +1,49 @@
+const fs = module.require("fs");
+const Discord = module.require('discord.js');
+const guilds = require("../data/guilds.json");
+const mutes = require("../data/mutes.json");
+
+module.exports.run = async (bot, message, args) => {
+    if (!(message.channel.type === "text")) return;
+    console.log("Removing roles...");
+    //import logChannel.
+    const logChannel = message.guild.channels.get(guilds[message.guild.id].logChannelID);
+    //check permissions.
+    if (!message.member.hasPermission("MANAGE_MESSAGES")) return console.log(`${message.author.username} attempted to remove roles without sufficient permissions!`);
+    //import target member from the message.
+    let target = message.mentions.members.first() || message.guild.members.get(args[0]);
+    // Get roles the user has
+    let listroles = target.roles.map(r => r.id);
+    // Makes sure it doesn't delete the iagree role
+    var getroleslekeepiagree = String(listroles).replace("391637024693813249",'');

```

Comment:
The whole point of a removeroles command is to make it so that the person in question loses access to all channels...